doc/related_projects.rst

```diff
@@ -179,6 +179,8 @@ Domain specific packages
 - `MSMBuilder <http://msmbuilder.org/>`_  Machine learning for protein
   conformational dynamics time series.
 
+- `scikit-surprise <http://surpriselib.com>`_ A scikit for recommender systems.
```

Comment:
Could you expand the description a bit more.

Maybe you could use your tagline here - "A Python library for building and analysing recommender systems inspired by scikit-learn API."